src/components/BPageNav/BPageNav.vue

```diff
@@ -0,0 +1,167 @@
+<template>
+  <nav :id="id">
+      <ul role="menu" aria-label="Page Navigation">
+        <li v-for="(link, index) in links" role ="menuitem" :key="index" :class="{ 'inline': !column }">
+          <a
+            :href="`#${link.href}`"
+            :aria-label="`${link.label} ${link.completed ? 'Completed' : 'Uncompleted'}`"
+            :class="{ 'inline': !column }"
+            @click.prevent.stop="jumpTo(link)"
+            @keyup.enter.prevent="jumpTo(link)"
+            @keydown.enter.prevent
+            @keyup.up="upHandler($event)"
```

Comment:
`Up` and `Down` arrow still cause the page to scroll. I don't remember if `stop` or `prevent` stops this, nor if it needs to be on `keyup` or `keydown`. We should document that.

Reply:
Fixed.  And you just need to prevent default on keydown